steal-qunit.js

```diff
@@ -111,7 +111,7 @@ define([
 		if (window.Testee && window.Testee.init) {
 			Testee.init();
 		}
-		QUnit.load();
+		QUnit.start();
```

Comment:
Same as above, can we support both here? If `QUnit.load` is a 1.x think we can conditionally check for it:

```js
if(typeof QUnit.load === "function") {
  QUnit.load();
} else {
  QUnit.start();
}
```